docs/src/capi/tutorial/getting-started.md

```diff
@@ -1 +1,196 @@
+Kubernetes as a management cluster:
+
+```sh
+sudo snap install k8s --classic --edge
+sudo k8s bootstrap
+sudo k8s status --wait-ready
+mkdir -p ~/.kube/
+sudo k8s config > ~/.kube/config
+```
+
+When setting up the management cluster, place its kubeconfig under
+`~/.kube/config` so other tools such as `clusterctl` can discover and interact
+with it.
+
+### Prepare the infrastructure provider
+
+Before generating a cluster, you need to configure the infrastructure provider.
+Each provider has its own prerequisites. Please follow the instructions
+for your provider:
+
+````{tabs}
+```{group-tab} AWS
+
+The AWS infrastructure provider requires the `clusterawsadm` tool to be
+installed:
+
+```sh
+curl -L https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases/download/v2.5.2/clusterawsadm-linux-amd64 -o clusterawsadm
+chmod +x clusterawsadm
+sudo mv clusterawsadm /usr/local/bin
+```
+
+`clusterawsadm` helps you bootstrapping the AWS environment that CAPI will use
+It will also create the necessary IAM roles for you.
+
+Start by setting up environment variables defining the AWS account to use, if
+these are not already defined:
+
+```sh
+export AWS_REGION=<your-region-eg-us-east-1>
+export AWS_ACCESS_KEY_ID=<your-access-key>
+export AWS_SECRET_ACCESS_KEY=<your-secret-access-key>
+```
+
+If you are using multi-factor authentication, you will also need:
+
+```sh
+export AWS_SESSION_TOKEN=<session-token>
+```
+
+`clusterawsadm` uses these details to create a [CloudFormation] stack in your
+AWS account with the correct [IAM] resources:
+
+```sh
+clusterawsadm bootstrap iam create-cloudformation-stack
+```
+
+The credentials need to be encoded and stored as a Kubernetes secret:
+
+```sh
+export AWS_B64ENCODED_CREDENTIALS=$(clusterawsadm bootstrap credentials encode-as-profile)
+```
+
+You are now all set to deploy the AWS CAPI infrastructure provider.
+
+```
+````
+
+### Initialise the management cluster
+
+To initialise the management cluster with the latest released version of the
+providers and the infrastructure of your choice:
+
+```sh
+clusterctl init --bootstrap ck8s --control-plane ck8s -i <infra-provider-of-choice>
+```
+
+### Generate a cluster spec manifest
+
+Once the bootstrap and control-plane controllers are up and running, you can
+apply the cluster manifests with the specifications of the cluster you want to
+provision.
+
+You can generate a cluster manifest for a selected set of commonly used
+infrastructures via templates provided by the Canonical Kubernetes team.
+Ensure you have initialized the desired infrastructure provider and fetch
+the Canonical Kubernetes provider repository:
+
+```sh
+git clone https://github.com/canonical/cluster-api-k8s
+```
+
+Review the list of variables needed for the cluster template:
+
+```sh
+cd cluster-api-k8s
+clusterctl generate cluster <cluster-name> --from ./templates/<infrastructure-provider>/cluster-template.yaml --list-variables
```

Comment:
maybe consider adding a 'CLUSTER_NAME=yourk8scluster' or whatever step and then you can replace all the <cluster-name> bits. I think it makes it just as obvious what's going on but with the advantage that people can just copy and use the commands without editing them all